src/Maths/Silk.NET.Maths/Simd/Simd.GetWithElement.cs

```diff
@@ -0,0 +1,91 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Just putting this here so I don't forget whether we went through with this change